README.md

```diff
@@ -45,6 +45,31 @@ about them:
 Similarly, you may need to `npm link path/to/your/js/sdk` in your `matrix-react-sdk`
 directory.
 
+Example of installation and link for vector:
+
+```
+mkdir ~/matrix
+cd ~/matrix
+git clone https://github.com/vector-im/vector-web.git
+cd ~/vector-web
+git checkout develop
```

Comment:
It would be better to insert the block that checks out matrix-react-sdk and npm links here, otherwise running npm install will install the matrix-react-sdk listed in package.json which you'll then overwrite. Plus the npm run build will need to happen after you've done all the npnm links.

Also, if we're giving a complete example, it should probably include matrix-js-sdk develop too.